python/pyspark/__init__.py

```diff
@@ -55,6 +55,12 @@
 from pyspark.accumulators import Accumulator, AccumulatorParam
 from pyspark.broadcast import Broadcast
 from pyspark.resourceinformation import ResourceInformation
+from pyspark.taskresourcerequest import TaskResourceRequest
+from pyspark.executorresourcerequest import ExecutorResourceRequest
+from pyspark.taskresourcerequests import TaskResourceRequests
+from pyspark.executorresourcerequests import ExecutorResourceRequests
+from pyspark.resourceprofilebuilder import ResourceProfileBuilder
+from pyspark.resourceprofile import ResourceProfile
```

Comment:
Can we group these into less modules? e.g.,
- `resourceprofile` and `resourceprofilebuilder` -> `resource_profile`
- `executorresourcerequest` and `executorresourcerequests` -> `resource_request`,
- `taskresourcerequest` and `taskresourcerequests` -> `resource_request`

Reply:
how about putting them all into a resource module that way its similar to the scala side with org.apache.spark.resource.  ?  Or were you talking about actually combining the files?

then assume here we coudl do from pyspark.resource import *

Reply:
+1!